app/angular/src/client/preview/angular/helpers.ts

```diff
@@ -75,7 +77,9 @@ const draw = (newModule: DynamicComponentType): void => {
   if (!platform) {
     insertDynamicRoot();
     try {
-      enableProdMode();
+      if (NODE_ENV !== 'development') {
```

Comment:
Won't this possibly throw a runtime error if NODE_ENV isn't set?

Reply:
I think it will

Reply:
If not defined, should it be handled as a production or development mode?

Reply:
New suggestion: https://github.com/storybooks/storybook/pull/6215/commits/5d76d4738069ffa56bcba9d4ec534b5697dc4a46#diff-65b1b66b4dca31f62fc0028943d19079R84